include/mbedtls/asn1.h

```diff
@@ -261,6 +262,29 @@ int mbedtls_asn1_get_int( unsigned char **p,
                           const unsigned char *end,
                           int *val );
 
+/**
+ * \brief       Retrieve an integer ASN.1 tag and its value.
+ *              Updates the pointer to immediately behind the full tag.
+ *
+ * \param p     On entry, \c *p points to the start of the ASN.1 element.
+ *              On successful completion, \c *p points to the first byte
+ *              beyond the ASN.1 element.
+ *              On error, the value of \c *p is undefined.
+ * \param end   End of data.
+ * \param val   On success, the parsed value.
+ *
+ * \return      0 if successful.
+ * \return      An ASN.1 error code if the input does not start with
+ *              a valid ASN.1 ENUMERATED.
+ * \return      #MBEDTLS_ERR_ASN1_INVALID_LENGTH if the parsed value does
+ *              not fit in an \c int.
+ * \return      An ASN.1 error code if the input does not start with
```

Comment:
This clause is present twice. That's a preexisting bug in the description of `mbedtls_asn1_get_int`. Can you please remove the second occurrence from the documentation of both functions?

Reply:
I can see this is present not only in mbedtls_asn1_get_int. Should I remove it everywhere in asn1.h?

Reply:
It's present for many functions, but it should only be present once per function.

Reply:
I understand this. But if you look at mbedtls_asn1_get_bitstring_null(...), mbedtls_asn1_get_sequence_of(...), mbedtls_asn1_get_bool(...), etc. - you will also find multiple includes of return statements. If we are talking about fixing docs for mbedtls_asn1_get_int() - should be all multiple returns fixed as part of this PR, or just leave it as is?

Reply:
It's ok to have multiple `\return` directives for the same function. That's our house style.

The problem with `mbedtls_asn1_get_int`, `mbedtls_asn1_get_mpi` and now `mbedtls_asn1_get_enum` is that they have multiple `\return` directives _with the same content_. That's noise.

(And probably my fault because I revised `asn1.h` recently. But since that's already merged and you've revealed it, please fix it.)

Reply:
Ah, got it. Was a bit confused because quoted code was not including repeating values. Ok, I will fix this.